.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: nzHjkNIQ3J4IgpiyCouoRAI6ILoTe3M4d
```

Comment:
this should not be exposed for everyone to see, instead, it should be a `.env ` variable